createst.py

```diff
@@ -141,10 +141,20 @@ def write_to_file(data):
         sys.exit(1)
     with open(test_yaml_path, "w+") as fp:
         fp.write("# *** Add configuration here ***\n\n")
-        if not args["strictcsums"]:
-            fp.write("args:\n- -k none\n\n")
+        if check_set_args():
```

Comment:
This will never be true for checksums and we'll end up not ever having `-k none` unless we have `--midstream` as an arg.